mirror/images.txt

```diff
@@ -7,7 +7,7 @@
 #
 
 # ceph-csi devel
-docker.io/rook/ceph:v1.10.4      rook/ceph:v1.10.4
+docker.io/rook/ceph:v1.10.8      rook/ceph:v1.10.8
```

Comment:
use 1.10.9 https://github.com/rook/rook/releases/tag/v1.10.9  as its the latest release